build/publish/PublishNupkgToBlobFeed.targets

```diff
@@ -7,13 +7,13 @@
     <NupkgsForPublishing Include="$(PackagesDirectory)/Microsoft.DotNet.MSBuildSdkResolver.*.nupkg" />
   </ItemGroup>
 
-  <Target Name="PublishNupkgToTransportFeed" Condition=" '$(PUBLISH_NUPKG_TO_TRANSPORT_FEED)' == 'true' AND '$(OS)' == 'Windows_NT' And '$(Architecture)' == 'x64' " >
-    <Error Condition="'$(TransportFeedCloudDropAccessToken)' == ''" Text="Missing property TransportFeedCloudDropAccessToken." />
-    <Error Condition="'$(TransportFeedExpectedURL)' == ''" Text="Missing property TransportFeedExpectedURL." />
+  <Target Name="PublishNupkgToBlobFeed" Condition=" '$(PUBLISH_NUPKG_TO_BLOB_FEED)' == 'true' AND '$(OS)' == 'Windows_NT' And '$(Architecture)' == 'x64' " >
```

Comment:
Does PUBLISH_NUPKG_TO_BLOB_FEED hook up to PB_PublishType somewhere?

Reply:
Just before merge, I will need to change "PUBLISH_NUPKG_TO_TRANSPORT_FEED" to "PUBLISH_NUPKG_TO_BLOB_FEED" in the VSO build definitions. See: https://devdiv.visualstudio.com/DevDiv/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=6786